### ISSUE TITLE
Communications link failure

### ISSUE THREAD

**Issue by ipozow** [AuthMe/AuthMeReloaded#2070]
### What behaviour is observed:
What happened?
Summary: LuckPerms was the problem but is more than that. 
Authme won't start because communications link failure. It happened after I did an apt-get update and apt-get upgrade on my server. The server was running when I did the upgrades, and when trying to join the server asked for registration, and also didn't allow to register if that was tried. I was using AuthMe v5.6.0-beta1-b2226 when this happened. Then I tried to start the plugin with the default config in a test server and didn't work. So I downloaded the dev build v5.6.0-SNAPSHOT-b2390 and tried with default config, and worked. But didn't work for the MySQL one. I have tested the database and is working fine. Then I searched a bit and tried to disable SSL and certificate check and now works fine but without LuckPerms. So the problem is LuckPerms
### What behaviour is expected:
What did you expect?
The plugin to run normal after the upgrades. By the way I am using Ubuntu 16.04

### Steps/models to reproduce:
The actions that cause the issue
Install AuthMe, connect to a MySQL database.
Install LuckPerms

But I think that is something related with the operating system because before it worked fine
### Plugin list:
This can be found by running `/pl`
Only in the test server AuthMe,
Normal server AuthMe + LuckPerms 
### Environment description
Standalone server/Bungeecord network, SQLite/MySql, ...

### AuthMe build number:
This can be found by running `/authme version`
v5.6.0-SNAPSHOT-b2390
### Error Log:
Pastebin/Hastebin/Gist link of the error log or stacktrace (if any)
SSL and mySQLCheckServerCertificate enabled-> https://pastebin.com/dJkEJQr2 
SSL and mySQLCheckServerCertificate disabled-> https://pastebin.com/BnjQgEL0
### Configuration:
Pastebin/Hastebin/Gist link of your config.yml file (remember to delete any sensitive data)
Is the default one but with the MySQL thing and passwords

**Comment by Vizis** [comment]
Yeah i came here with same issue, lastes version of LP (5.0.72) is crashing authme. Authme with version  5.4 adn 5.6 is crashing. Authme connected to mysql w/o problem after adding lp server can run cause of >> [AuthMe] Aborting initialization of AuthMe: [InjectorReflectionException]: Could not invoke method 'setup' for fr.xephi.authme.permission.PermissionsManager@
